README.md

```diff
@@ -1 +1,50 @@
 The goal of this project is to explore the network properties of T-cell repertoires.
+# code structure
+
+```
+net
+├─ singleNet.py
+├─ manyNets.py
+├─ levenshtein.py
+├─ jaccard.py
+├─ funcDictionarySingleNet.py
+├─ funcDictionaryManyNets.py
+├─ funcDictionaryLevenshtein.py
+├─ funcDictionaryJaccard.py
+├─ funcDictionary.py
+└─ plotData.py
+```
+
+This repository is still under construction and not yet finished. Please do not yet use it.
+
+The files singleNet.py, manyNets.py, levenshtein.py, jaccard.py perform network analysis and save them as txt files. 
```

Comment:
You can use `' to mark files. Instead of singleNet.py you would have `singleNet.py`.